packages/cdk8s-cli/lib/import/base.ts

```diff
@@ -85,6 +85,14 @@ export abstract class ImportBase {
             };
           }
 
+          // java!
+          if (options.targetLanguage === Language.JAVA) {
+            opts.java = {
+              outdir: outdir,
+              package: moduleNamePrefix ? `${moduleNamePrefix}.${name}` : name
```

Comment:
So something like:

`import k8s.Service` becomes `import hello.k8s.Service`. You'd have to make changes to your `pom.xml` file as well.